### ISSUE TITLE
You can ceiling-bug again for some reason

### ISSUE THREAD

**Issue by Ninja-Cookie** [momentum-mod/game#1622]
### Describe the bug

By jumping into a ceiling with a trigger above it and pressing crouch as you reach the ceiling, you are able to touch the trigger through the ceiling and activate it.

https://user-images.githubusercontent.com/62808028/146709483-a04a4e10-9302-4725-b8ec-65f98b3b06ab.mp4

### How To Reproduce

1. Jump towards a ceiling with a trigger low above it
2. Crouch as you touch the ceiling

### Expected Behavior

Don't activate the trigger.

### Operating System

Windows

### Renderer

DX11 (default)

**Comment by jason-e** [comment]
The fix for this bug (`sv_duck_collision_fix`) is not enabled on RJ/SJ by default.

**Comment by Gocnak** [comment]
Yep it's intentionally off for RJ/SJ as it was explicitly requested to be so. Whether it should be fixed for these modes is a community discussion to be had.

**Comment by hexaflexahexagon** [comment]
i started a discussion about this a few days ago in the rj channel and it was pretty split, some people couldn't care less while other people liked them and didn't want them to be removed. my personal opinion is that most people like them because of being able to do skips (similar to bounces) which aren't going to be relevant in mmod due to having to hit every zone by default to submit a run. because of that i personally would like them removed, it makes mappers lives easier and removes one more weird quirk that only RJ is leaving in, headbugs themselves are so easy it doesn't matter, you have like a 10 tick window which is impossible to fail. the 2 maps that use headbugs intentionally can replicate them using triggers very easily.
